app/src/utils/Utils.tsx

```diff
@@ -510,3 +510,7 @@ export const setMessageSnackbar = (message: string, context: IDialogContext) =>
  * @returns T
  */
 export const firstOrNull = <T,>(arr: T[]): T | null => (arr.length > 0 ? arr[0] : null);
+
+export const isMeasurementQualitative = (): boolean => {
```

Comment:
Might want a comment to justify the return value here

Reply:
looks like its unused, can remove